### DISCUSSION TITLE
How to install terrakube on AWS EKS Cluster

### DISCUSSION THREAD

**Discussion by Udayasankar6** [terrakube-io/terrakube#838] [Q&A]
I am trying to install Terrakube on EKS cluster on AWS but I could not find any proper documentation around it.
Could anyone help me if anyone has already done this successfully? 
what all resources would I need to successfully install and view the terrakube ui or use api

**Comment by alfespa17** [comment]
You could check the examples.

https://github.com/AzBuilder/terrakube-helm-chart/tree/main/examples

In a high level you new to create your dex configuration and the ingress setup, you can find information here too

https://docs.terrakube.io/getting-started/deployment/ingress-configuration

https://docs.terrakube.io/getting-started/deployment/user-authentication-dex

**Reply by Udayasankar6** [reply]
Hey @alfespa17  Thankyou! for the response,
to add on to my question I am building a sample POC only first, can i use the domains/subdomains  terrakube-ui.minikube.net, terrakube-api.minikube.net, terrakube-reg.minikube.net 
and also do you have some predifined terraform module or any other way of giving a required configuration for ALB and EKS cluster?

**Reply by alfespa17** [reply]
You won't be able to use that domains, those are just for testing purposes when you deployed terrakube in minikube as you can see here

https://docs.terrakube.io/getting-started/deployment/minikube-+-https

You can't use those domains because if you installed in a real cluster you will need to update your dns provider with your own domains, I have no experience with AWS so I can't help with the ingress settings for that configuration.

**Reply by Udayasankar6** [reply]
Hey @alfespa17 Thankyou again! for quick responses and updates. If you know someone else who could help us with the aws setup and dex config could you point me to them?

And also just a suggestion, it would be better if we could get some 1 click setup for terrakube. and also easy way to get a proper documentation on getting started. So that user dont have to worry about configuring so many things before they can actually start testing terrakube with terraform.
